ajax/kanban.php

```diff
@@ -89,7 +89,7 @@
             return;
         }
     }
-    if (in_array($action, ['add_teammember'])) {
+    if (in_array($action, ['load_teammember_form', 'add_teammember'])) {
```

Comment:
Is it really necessary to check the action ```load_teammember_form``` here?

As verified/used later (L301)

Reply:
This is used here to load the item